src/main/java/org/gitlab4j/api/MergeRequestApi.java

```diff
@@ -279,6 +280,44 @@ public MergeRequest getMergeRequest(Integer projectId, Integer mergeRequestIid)
                 "projects", projectId, "merge_requests", mergeRequestIid, "commits"));
     }
 
+    /**
```

Comment:
The discussions related methods should be moved to a class called ```DiscussionApi```.  This library follows a pattern where we add an API class for each API described in the GitLab API documentation.  This one is the "Discussions API" as described here: https://docs.gitlab.com/ee/api/discussions.html